lint-yaml/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: Lint YAML
+description: run yamllint with reviewdog
+
+inputs:
+  submodules:
+    description: true. false or recursive
+    required: false
+    default: 'false'
+
+runs:
+  using: composite
+  steps:
+  - name: Checkout
+    uses: mobilecoinofficial/gh-actions/checkout@v0
+    with:
+      submodules: ${{ inputs.submodules }}
+
+  - name: Run yamllint with reviewdog
+    uses: reviewdog/action-yamllint@f01d8a48fd8d89f89895499fca2cff09f9e9e8c0
```

Comment:
Renovate can track this if you add a comment and a version that this hash references at the end of the line, like this:

`uses: reviewdog/action-yamllint@f01d8a48fd8d89f89895499fca2cff09f9e9e8c0 # v1.3.0`

See: https://docs.renovatebot.com/modules/manager/github-actions/